packages/design-system-tokens/src/themes/medicare-components.ts

```diff
@@ -419,6 +419,18 @@ export const components: AnyTokenValues = {
     '-trigger__color':                            t.color['base'],
   },
 
+  'typography': {
+    '-heading__font-family':                      t.font['family-montserrat'],
+    '-heading__font-weight':                      t.font['weight-semibold'],
+    '-heading-4xl__font-weight':                  t.font['weight-light'],
+    '-heading-3xl__font-weight':                  t.font['weight-semibold'],
+    '-heading-lg__font-weight':                   t.font['weight-semibold'],
+    '-heading-md__font-weight':                   t.font['weight-semibold'],
+    '-heading-sm__font-weight':                   t.font['weight-semibold'],
+    '-body__color':                               t.color['base'],
```

Comment:
It looks like `t.color['base']` is the same value used for the color in `.ds-base`, so I'm wondering if this can be `inherit` like the other themes given [the change found here](https://github.com/CMSgov/design-system/pull/2079/files#r960109260). The color of the drawer footer content on master might be a bug, though.

Reply:
The problem I see with setting this to `base `explicitly here is that if we're using _on dark_ style in a parent element, this won't pick up on that automatically. Not sure if that's an actual problem, though.